### PR TITLE
chore: excluding java-maps-* modules

### PR DESCRIPTION
**Pull request by suztomo**
GroupId needs to be discussed further.

### CODE REVIEW

**Review by meltsufin — CHANGES_REQUESTED** [review]
@suztomo Can you explain why?
This is going to skip the entire module when doing operations from the project root.

**Comment by meltsufin** [comment]
@suztomo Can you just add skip deploy config?

**Comment by suztomo** [comment]
@meltsufin I prefer removing the module from the root to figuring out the proper configuration to exclude. It's simpler than modifying pom.xml files. As you know Maven nexus deploy configuration is not simple. (Question about "-bom", "grpc-", and "proto-" artifacts).

File: generation/print_root_pom.sh [hunk 1]
```diff
@@ -8,4 +8,6 @@ GENERATION_DIR=$(dirname -- "$0");
 find . -mindepth 2 -maxdepth 2 -name pom.xml |sort --dictionary-order | xargs dirname \
     |sed -e 's|./||' | xargs -I '{}' echo "    <module>{}</module>" > /tmp/repo-modules.txt
 
+sed -i.bak '/java-maps/d' /tmp/repo-modules.txt
```
Comment (meltsufin) [inline generation/print_root_pom.sh:11]: No need for this anymore, right?

Comment (suztomo) [reply generation/print_root_pom.sh]: removed.

**Pull request merged**